modules/nw-dpu-installing-operator.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/installing-dpu-operator.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-dpu-operator_{context}"]
+= Installing the DPU Operator
+
+The DPU Operator on the host cluster side is responsible for the life-cycle management of the `ovn-kube` components and the necessary host network initialization on IPU.
```

Comment:
The DPU Operator is an operator that should be installed on both the host side cluster and all the DPU side clusters. It is responsible for the life-cycle management of all the supported DPUs and takes care. The user should not interact with the DPU manually.